mypy/build.py

```diff
@@ -602,6 +602,12 @@ def __init__(self, data_dir: str,
         self.fscache = fscache
         self.find_module_cache = FindModuleCache(self.search_paths, self.fscache, self.options)
         self.metastore = create_metastore(options)
+        if fscache.isdir(options.cache_dir):
```

Comment:
I would suggest adding the `.gitignore` file only on creating the cache directory, as I wouldn't be surprised if some people store it in git intentionally. With this they would have to delete the .gitignore file every run.

Reply:
Sure, why not, 90013eb should take care of it. Also using `os.path.isdir` instead of `fscache.isdir` as metastore creates dirs without updating fscache's stat cache.